src/lib-components/layout/SidebarLayout.vue

```diff
@@ -116,6 +116,11 @@ const isActive = (url: string): boolean => {
                   {{ item.name }}
                 </a>
               </nav>
+              <div v-if="$slots['sidebar-bottom']" class="mt-auto">
```

Comment:
It can look a touch cramped under certain scenarios, but the horizontal padding is set to be consistent with the primary nav, which I think makes sense to avoid being too prescriptive about the layout of the component that falls in that slot.  It'll be easy enough to add additional spacing as needed.